plugins/nonpersistent_voxel_layer.cpp

```diff
@@ -118,15 +118,13 @@ void NonPersistentVoxelLayer::matchSize()
 
 void NonPersistentVoxelLayer::reset()
 {
-  deactivate();
+  ObstacleLayer::reset();
   resetMaps();
-  voxel_grid_.reset();
```

Comment:
This is strictly required, else the voxel grid won't be reset.

Reply:
reset was being done twice.  it remains in `resetMaps`